tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -0,0 +1,540 @@
+| VisualShaderNodeVectorScalarSmoothStep  | VisualShaderNodeSmoothStep                |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeVectorScalarStep        | VisualShaderNodeStep                      |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeScalarSwitch            | VisualShaderNodeSwitch                    |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeScalarTransformMult     | VisualShaderNodeTransformOp               |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeScalarDerivativeFunc    | VisualShaderNodeDerivativeFunc            |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeVectorDerivativeFunc    | VisualShaderNodeDerivativeFunc            |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeBooleanUniform          | VisualShaderNodeBooleanParameter          |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeColorUniform            | VisualShaderNodeColorParameter            |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeScalarUniform           | VisualShaderNodeFloatParameter            |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeCubeMapUniform          | VisualShaderNodeCubeMapParameter          |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeTextureUniform          | VisualShaderNodeTexture2DParameter        |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeTextureUniformTriplanar | VisualShaderNodeTextureParameterTriplanar |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeTransformUniform        | VisualShaderNodeTransformParameter        |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeVec3Uniform             | VisualShaderNodeVec3Parameter             |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeUniform                 | VisualShaderNodeParameter                 |
++-----------------------------------------+-------------------------------------------+
+| VisualShaderNodeUniformRef              | VisualShaderNodeParameterRef              |
++-----------------------------------------+-------------------------------------------+
+
+If you cannot find a node or resource in the list below, refer to the above
+table to find its new name.
```

Comment:
The meaning of this sentence isn't really clear IMO. As a reader I don't know yet what "the list below" is, so it's confusing to be told to check the table above to find things if I can't find them...